prog/vm/nexus.rb

```diff
@@ -188,6 +189,7 @@ def before_run
     queued_vms = Vm.join(:strand, id: :id).where(:location => vm.location, :arch => vm.arch, Sequel[:strand][:label] => "start")
     begin
       distinct_storage_devices = frame["distinct_storage_devices"] || false
+      host_exclusion_filter = frame["exclude_host_ids"] || []
```

Comment:
Should we add a check and error out if `exclude_host_ids` cover `force_host_id`?

Reply:
you think maybe `fail BUG:` type fidelity of error?

Reply:
If `exclude_host_ids` and `force_host_id` intersect, we'd get a "no space left on any eligible host" error right now. We can fail earlier with such a specific error.